hadoop-hdds/docs/content/shell/BucketCommands.md

```diff
@@ -26,7 +26,6 @@ Ozone shell supports the following bucket commands.
   * [delete](#delete)
   * [info](#info)
   * [list](#list)
-  * [update](#update)
```

Comment:
Should we block the acl update part of BucketManagerImpl#setBucketProperty() as they now require a different permission (WRITE_ACL instead of WRITE)? I see there are  few UT assumes setBucketProperty should be able to change acl without differentiation. We can fix his in follow up JIRA and use the current one just to remove the CLI.

Reply:
Ya sure @xiaoyuyao.
I will open a new Jira to address this too.
We can let this change in to fix only remove CLI part.

Reply:
https://issues.apache.org/jira/browse/HDDS-1913
Will address this and also fixing Bucket and RpcClient API's.

Reply:
Ping @xiaoyuyao for review.

Reply:
LGTM. +1. Let's fix the remaining issues in HDDS-1913.